protocols/virtualdefund/virtualdefund.go

```diff
@@ -17,14 +17,15 @@ import (
 )
 
 const (
-	WaitingForCompleteFinal           protocols.WaitingFor = "WaitingForCompleteFinal"           // Round 1
+	WaitingForFinalStateFromAlice     protocols.WaitingFor = "WaitingForFinalStateFromAlice"
+	WaitingForSignedFinal             protocols.WaitingFor = "WaitingForSignedFinal"             // Round 1
 	WaitingForCompleteLedgerDefunding protocols.WaitingFor = "WaitingForCompleteLedgerDefunding" // Round 2
 	WaitingForNothing                 protocols.WaitingFor = "WaitingForNothing"                 // Finished
 )
 
 const (
-	SignedStatePayload protocols.PayloadType = "SignedStatePayload"
-	InitPayload        protocols.PayloadType = "InitPayload"
+	SignedStatePayload       protocols.PayloadType = "SignedStatePayload"
+	FinalStateRequestPayload protocols.PayloadType = "FinalStateRequestPayload"
```

Comment:
Might be worth some godoc on these types?

Reply:
Is `RequestFinalStatePayload` better?

Reply:
Added some godoc and renamed in [c0aaaee](https://github.com/statechannels/go-nitro/pull/866/commits/c0aaaee3408ae1e49ccb6acd088549047b452013)